### ISSUE TITLE
Setup fails to take down Realtek Audio Universal Service kernel module if usermode registration is missing

### ISSUE THREAD

**Issue by pal1000** [pal1000/Realtek-UAD-generic#20]
@julianxhokaxhiu I need to collect some logs to figure out why the service can't be terminated on your computer:
- screenshot(s) from [Driverstore Explorer](https://github.com/lostindark/DriverStoreExplorer/releases);
- the output of this command in powershell: `(get-wmiobject -Query 'Select * from win32_service') | Select-object Name, Pathname`.

**Comment by julianxhokaxhiu** [comment]
### Screenshots
![image](https://user-images.githubusercontent.com/1237070/68692566-ff413f00-0575-11ea-802d-1e63346dd52c.png)
![image](https://user-images.githubusercontent.com/1237070/68692586-06684d00-0576-11ea-9590-463ec46dcc3b.png)
![image](https://user-images.githubusercontent.com/1237070/68692593-0bc59780-0576-11ea-9f1b-913be6eb803a.png)

### PowerShell Output
```
PS C:\Users\Julian> (get-wmiobject -Query 'Select * from win32_service') | Select-object Name, Pathname                                                                                                                                                                        
Name                                                   Pathname
----                                                   --------
AdobeARMservice                                        "C:\Program Files (x86)\Common Files\Adobe\ARM\1.0\armsvc.exe"
AJRouter                                               C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
ALG                                                    C:\Windows\System32\alg.exe
AppIDSvc                                               C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
Appinfo                                                C:\Windows\system32\svchost.exe -k netsvcs -p
AppMgmt                                                C:\Windows\system32\svchost.exe -k netsvcs -p
AppReadiness                                           C:\Windows\System32\svchost.exe -k AppReadiness -p
AppVClient                                             C:\Windows\system32\AppVClient.exe
AppXSvc                                                C:\Windows\system32\svchost.exe -k wsappx -p
AssignedAccessManagerSvc                               C:\Windows\system32\svchost.exe -k AssignedAccessManagerSvc
AudioEndpointBuilder                                   C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
Audiosrv                                               C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
autotimesvc                                            C:\Windows\system32\svchost.exe -k autoTimeSvc
AxInstSV                                               C:\Windows\system32\svchost.exe -k AxInstSVGroup
BDESVC                                                 C:\Windows\System32\svchost.exe -k netsvcs -p
BFE                                                    C:\Windows\system32\svchost.exe -k LocalServiceNoNetworkFirewall -p
BITS                                                   C:\Windows\System32\svchost.exe -k netsvcs -p
BrokerInfrastructure                                   C:\Windows\system32\svchost.exe -k DcomLaunch -p
Browser                                                C:\Windows\System32\svchost.exe -k netsvcs -p
BTAGService                                            C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted
BthAvctpSvc                                            C:\Windows\system32\svchost.exe -k LocalService -p
bthserv                                                C:\Windows\system32\svchost.exe -k LocalService -p
camsvc                                                 C:\Windows\system32\svchost.exe -k appmodel -p
CDPSvc                                                 C:\Windows\system32\svchost.exe -k LocalService -p
CertPropSvc                                            C:\Windows\system32\svchost.exe -k netsvcs
ClickToRunSvc                                          "C:\Program Files\Common Files\Microsoft Shared\ClickToRun\OfficeClickToRun.exe" /service
ClipSVC                                                C:\Windows\System32\svchost.exe -k wsappx -p
COMSysApp                                              C:\Windows\system32\dllhost.exe /Processid:{02D4B3F1-FD88-11D1-960D-00805FC79235}
CoreMessagingRegistrar                                 C:\Windows\system32\svchost.exe -k LocalServiceNoNetwork -p
cphs                                                   C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_55ea1c9e3c7e8c44\IntelCpHeciSvc.exe
cplspcon                                               C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_55ea1c9e3c7e8c44\IntelCpHDCPSvc.exe
CryptSvc                                               C:\Windows\system32\svchost.exe -k NetworkService -p
CscService                                             C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
DcomLaunch                                             C:\Windows\system32\svchost.exe -k DcomLaunch -p
defragsvc                                              C:\Windows\system32\svchost.exe -k defragsvc
DeviceAssociationService                               C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
DeviceInstall                                          C:\Windows\system32\svchost.exe -k DcomLaunch -p
DevQueryBroker                                         C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
Dhcp                                                   C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
diagnosticshub.standardcollector.service               C:\Windows\system32\DiagSvcs\DiagnosticsHub.StandardCollector.Service.exe
diagsvc                                                C:\Windows\System32\svchost.exe -k diagnostics
DiagTrack                                              C:\Windows\System32\svchost.exe -k utcsvc -p
DispBrokerDesktopSvc                                   C:\Windows\system32\svchost.exe -k LocalService -p
DisplayEnhancementService                              C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
DmEnrollmentSvc                                        C:\Windows\system32\svchost.exe -k netsvcs -p
dmwappushservice                                       C:\Windows\system32\svchost.exe -k netsvcs -p
Dnscache                                               C:\Windows\system32\svchost.exe -k NetworkService -p
DoSvc                                                  C:\Windows\System32\svchost.exe -k NetworkService -p
dot3svc                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
DPS                                                    C:\Windows\System32\svchost.exe -k LocalServiceNoNetwork -p
DSAService                                             "C:\Program Files (x86)\Intel\Driver and Support Assistant\DSAService.exe"
DSAUpdateService                                       "C:\Program Files (x86)\Intel\Driver and Support Assistant\DSAUpdateService.exe"
DsmSvc                                                 C:\Windows\system32\svchost.exe -k netsvcs -p
DsSvc                                                  C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
DusmSvc                                                C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
Eaphost                                                C:\Windows\System32\svchost.exe -k netsvcs -p
EFS                                                    C:\Windows\System32\lsass.exe
embeddedmode                                           C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
EntAppSvc                                              C:\Windows\system32\svchost.exe -k appmodel -p
esifsvc                                                C:\Windows\System32\Intel\DPTF\esif_uf.exe
ESRV_SVC_QUEENCREEK                                    "C:\Program Files\Intel\SUR\QUEENCREEK\x64\esrv_svc.exe" "--AUTO_START" "--start" "--start_options_registry_key" "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\ESRV_SVC_QUEENCREEK\_start"
EventLog                                               C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
EventSystem                                            C:\Windows\system32\svchost.exe -k LocalService -p
Fax                                                    C:\Windows\system32\fxssvc.exe
fdPHost                                                C:\Windows\system32\svchost.exe -k LocalService -p
FDResPub                                               C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation -p
fhsvc                                                  C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
FontCache                                              C:\Windows\system32\svchost.exe -k LocalService -p
FontCache3.0.0.0                                       C:\Windows\Microsoft.Net\Framework64\v3.0\WPF\PresentationFontCache.exe
FrameServer                                            C:\Windows\System32\svchost.exe -k Camera
GalaxyClientService                                    "C:\Program Files (x86)\GOG Galaxy\GalaxyClientService.exe"
GalaxyCommunication                                    "C:\ProgramData\GOG.com\Galaxy\redists\GalaxyCommunication.exe"
GoogleChromeElevationService                           "C:\Program Files (x86)\Google\Chrome\Application\78.0.3904.97\elevation_service.exe"
gpsvc                                                  C:\Windows\system32\svchost.exe -k netsvcs -p
GraphicsPerfSvc                                        C:\Windows\System32\svchost.exe -k GraphicsPerfSvcGroup
gupdate                                                "C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /svc
gupdatem                                               "C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /medsvc
HfcDisableService                                      C:\Windows\System32\DriverStore\FileRepository\iastorac.inf_amd64_0c50c5dc47ed0efe\HfcDisableService.exe
hidserv                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
hpqcaslwmiex                                           "C:\Program Files (x86)\HP\Shared\hpqwmiex.exe"
HPSupportSolutionsFrameworkService                     "C:\Program Files (x86)\Hewlett-Packard\HP Support Solutions\HPSupportSolutionsFrameworkService.exe"
HvHost                                                 C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
iaStorAfsService                                       C:\Windows\System32\iaStorAfsService.exe
IAStorDataMgrSvc                                       "C:\Program Files\Intel\Intel(R) Rapid Storage Technology\IAStorDataMgrSvc.exe"
icssvc                                                 C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
igfxCUIService2.0.0.0                                  C:\Windows\System32\DriverStore\FileRepository\cui_dch.inf_amd64_e7523682cc7528cc\igfxCUIService.exe
IKEEXT                                                 C:\Windows\system32\svchost.exe -k netsvcs -p
InstallService                                         C:\Windows\System32\svchost.exe -k netsvcs -p
Intel(R) Capability Licensing Service TCP IP Interface "C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS\SocketHeciServer.exe"
Intel(R) SUR QC SAM                                    "C:\Program Files\Intel\SUR\QUEENCREEK\Updater\bin\IntelSoftwareAssetManagerService.exe"
Intel(R) TPM Provisioning Service                      "C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS\TPMProvisioningService.exe"
iphlpsvc                                               C:\Windows\System32\svchost.exe -k NetSvcs -p
IpxlatCfgSvc                                           C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
jhi_service                                            "C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL\jhi_service.exe"
KeyIso                                                 C:\Windows\system32\lsass.exe
KtmRm                                                  C:\Windows\System32\svchost.exe -k NetworkServiceAndNoImpersonation -p
LanmanServer                                           C:\Windows\system32\svchost.exe -k netsvcs -p
LanmanWorkstation                                      C:\Windows\System32\svchost.exe -k NetworkService -p
lfsvc                                                  C:\Windows\system32\svchost.exe -k netsvcs -p
LicenseManager                                         C:\Windows\System32\svchost.exe -k LocalService -p
lltdsvc                                                C:\Windows\System32\svchost.exe -k LocalService -p
lmhosts                                                C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
LMS                                                    "C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\LMS\LMS.exe"
LSM
LxpSvc                                                 C:\Windows\system32\svchost.exe -k netsvcs
MapsBroker                                             C:\Windows\System32\svchost.exe -k NetworkService -p
MozillaMaintenance                                     "C:\Program Files (x86)\Mozilla Maintenance Service\maintenanceservice.exe"
mpssvc                                                 C:\Windows\system32\svchost.exe -k LocalServiceNoNetworkFirewall -p
MSDTC                                                  C:\Windows\System32\msdtc.exe
MSiSCSI                                                C:\Windows\system32\svchost.exe -k netsvcs -p
msiserver                                              C:\Windows\system32\msiexec.exe /V
NaturalAuthentication                                  C:\Windows\system32\svchost.exe -k netsvcs -p
NcaSvc                                                 C:\Windows\System32\svchost.exe -k NetSvcs -p
NcbService                                             C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
NcdAutoSetup                                           C:\Windows\System32\svchost.exe -k LocalServiceNoNetwork -p
Netlogon                                               C:\Windows\system32\lsass.exe
Netman                                                 C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
netprofm                                               C:\Windows\System32\svchost.exe -k LocalService -p
NetSetupSvc
NetTcpPortSharing                                      C:\Windows\Microsoft.NET\Framework64\v4.0.30319\SMSvcHost.exe
NgcCtnrSvc                                             C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
NgcSvc                                                 C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
NlaSvc                                                 C:\Windows\System32\svchost.exe -k NetworkService -p
nsi                                                    C:\Windows\system32\svchost.exe -k LocalService -p
NvContainerLocalSystem                                 "C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerLocalSystem -f "C:\ProgramData\NVIDIA\NvContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plugins\L...
NvContainerNetworkService                              "C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerNetworkService -f "C:\ProgramData\NVIDIA\NvContainerNetworkService.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plu...
NVDisplay.ContainerLocalSystem                         "C:\Program Files\NVIDIA Corporation\Display.NvContainer\NVDisplay.Container.exe" -s NVDisplay.ContainerLocalSystem -f "C:\ProgramData\NVIDIA\NVDisplay.ContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA C...
Origin Client Service                                  "C:\Program Files (x86)\Origin\OriginClientService.exe"
Origin Web Helper Service                              "C:\Program Files (x86)\Origin\OriginWebHelperService.exe"
ose64                                                  "c:\Program Files\Common Files\Microsoft Shared\Source Engine\OSE.EXE"
p2pimsvc                                               C:\Windows\System32\svchost.exe -k LocalServicePeerNet
p2psvc                                                 C:\Windows\System32\svchost.exe -k LocalServicePeerNet
PcaSvc                                                 C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
PeerDistSvc                                            C:\Windows\System32\svchost.exe -k PeerDist
perceptionsimulation                                   C:\Windows\system32\PerceptionSimulation\PerceptionSimulationService.exe
PerfHost                                               C:\Windows\SysWow64\perfhost.exe
PhoneSvc                                               C:\Windows\system32\svchost.exe -k LocalService -p
pla                                                    C:\Windows\System32\svchost.exe -k LocalServiceNoNetwork -p
PlugPlay                                               C:\Windows\system32\svchost.exe -k DcomLaunch -p
PNRPAutoReg                                            C:\Windows\System32\svchost.exe -k LocalServicePeerNet
PNRPsvc                                                C:\Windows\System32\svchost.exe -k LocalServicePeerNet
PolicyAgent                                            C:\Windows\system32\svchost.exe -k NetworkServiceNetworkRestricted -p
Power                                                  C:\Windows\system32\svchost.exe -k DcomLaunch -p
PrintNotify                                            C:\Windows\system32\svchost.exe -k print
ProfSvc                                                C:\Windows\system32\svchost.exe -k netsvcs -p
PushToInstall                                          C:\Windows\System32\svchost.exe -k netsvcs -p
QWAVE                                                  C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation -p
RasAuto                                                C:\Windows\System32\svchost.exe -k netsvcs -p
RasMan                                                 C:\Windows\System32\svchost.exe -k netsvcs
RemoteAccess                                           C:\Windows\System32\svchost.exe -k netsvcs
RemoteRegistry                                         C:\Windows\system32\svchost.exe -k localService -p
RetailDemo                                             C:\Windows\System32\svchost.exe -k rdxgroup
RmSvc                                                  C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted
Rockstar Service                                       "C:\Program Files\Rockstar Games\Launcher\RockstarService.exe"
RpcEptMapper                                           C:\Windows\system32\svchost.exe -k RPCSS -p
RpcLocator                                             C:\Windows\system32\locator.exe
RpcSs                                                  C:\Windows\system32\svchost.exe -k rpcss -p
RstMwService                                           C:\Windows\System32\DriverStore\FileRepository\iastorac.inf_amd64_0c50c5dc47ed0efe\RstMwService.exe
RtkAudioUniversalService                               "C:\Windows\System32\RtkAudUService64.exe"
RtkBtManServ                                           C:\Windows\RtkBtManServ.exe
SamSs                                                  C:\Windows\system32\lsass.exe
SCardSvr                                               C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation
ScDeviceEnum                                           C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted
Schedule                                               C:\Windows\system32\svchost.exe -k netsvcs -p
SCPolicySvc                                            C:\Windows\system32\svchost.exe -k netsvcs
SDRSVC                                                 C:\Windows\system32\svchost.exe -k SDRSVC
seclogon                                               C:\Windows\system32\svchost.exe -k netsvcs -p
SecurityHealthService                                  C:\Windows\system32\SecurityHealthService.exe
SEMgrSvc                                               C:\Windows\system32\svchost.exe -k LocalService -p
SENS                                                   C:\Windows\system32\svchost.exe -k netsvcs -p
Sense                                                  "C:\Program Files\Windows Defender Advanced Threat Protection\MsSense.exe"
SensorDataService                                      C:\Windows\System32\SensorDataService.exe
SensorService                                          C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
SensrSvc                                               C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation -p
SessionEnv                                             C:\Windows\System32\svchost.exe -k netsvcs -p
SgrmBroker                                             C:\Windows\system32\SgrmBroker.exe
SharedAccess                                           C:\Windows\System32\svchost.exe -k netsvcs -p
SharedRealitySvc                                       C:\Windows\system32\svchost.exe -k LocalService -p
ShellHWDetection                                       C:\Windows\System32\svchost.exe -k netsvcs -p
shpamsvc                                               C:\Windows\System32\svchost.exe -k netsvcs -p
smphost                                                C:\Windows\System32\svchost.exe -k smphost
SmsRouter                                              C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
SNMPTRAP                                               C:\Windows\System32\snmptrap.exe
SonosLibraryService                                    "C:\Program Files (x86)\Sonos\SonosLibraryService.exe"
spectrum                                               C:\Windows\system32\spectrum.exe
Spooler                                                C:\Windows\System32\spoolsv.exe
sppsvc                                                 C:\Windows\system32\sppsvc.exe
SSDPSRV                                                C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation -p
ssh-agent                                              C:\Windows\System32\OpenSSH\ssh-agent.exe
SstpSvc                                                C:\Windows\system32\svchost.exe -k LocalService -p
StateRepository                                        C:\Windows\system32\svchost.exe -k appmodel -p
Steam Client Service                                   "C:\Program Files (x86)\Common Files\Steam\SteamService.exe" /RunAsService
stisvc                                                 C:\Windows\system32\svchost.exe -k imgsvc
StorSvc                                                C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
svsvc                                                  C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
swprv                                                  C:\Windows\System32\svchost.exe -k swprv
SynaAPOService                                         "C:\Windows\System32\SynAudSrv.exe"
SynaAudioService                                       "C:\Windows\System32\CxAudioSvc.exe"
SynTPEnhService                                        C:\Windows\System32\SynTPEnhService.exe
SysMain                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
SystemEventsBroker                                     C:\Windows\system32\svchost.exe -k DcomLaunch -p
SystemUsageReportSvc_QUEENCREEK                        "C:\Program Files\Intel\SUR\QUEENCREEK\SurSvc.exe"
TabletInputService                                     C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
TapiSrv                                                C:\Windows\System32\svchost.exe -k NetworkService -p
TeamViewer                                             "C:\Program Files (x86)\TeamViewer\TeamViewer_Service.exe"
TermService                                            C:\Windows\System32\svchost.exe -k NetworkService
Themes                                                 C:\Windows\System32\svchost.exe -k netsvcs -p
TieringEngineService                                   C:\Windows\system32\TieringEngineService.exe
TimeBrokerSvc                                          C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
TokenBroker                                            C:\Windows\system32\svchost.exe -k netsvcs -p
TrkWks                                                 C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
TroubleshootingSvc                                     C:\Windows\system32\svchost.exe -k netsvcs -p
TrustedInstaller                                       C:\Windows\servicing\TrustedInstaller.exe
tzautoupdate                                           C:\Windows\system32\svchost.exe -k LocalService -p
UevAgentService                                        C:\Windows\system32\AgentService.exe
UmRdpService                                           C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
upnphost                                               C:\Windows\system32\svchost.exe -k LocalServiceAndNoImpersonation -p
UserManager                                            C:\Windows\system32\svchost.exe -k netsvcs -p
USER_ESRV_SVC_QUEENCREEK                               "C:\Program Files\Intel\SUR\QUEENCREEK\x64\esrv_svc.exe" "--run_as_user_process"
UsoSvc                                                 C:\Windows\system32\svchost.exe -k netsvcs -p
VacSvc                                                 C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
VaultSvc                                               C:\Windows\system32\lsass.exe
vds                                                    C:\Windows\System32\vds.exe
ViGEmBusUpdater                                        "C:\Program Files\Nefarius Software Solutions\ViGEm Bus Driver\ViGEmBusUpdater.exe" /runservice
VMAuthdService                                         "C:\Program Files (x86)\VMware\VMware Player\vmware-authd.exe"
vmicguestinterface                                     C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
vmicheartbeat                                          C:\Windows\system32\svchost.exe -k ICService -p
vmickvpexchange                                        C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
vmicrdv                                                C:\Windows\system32\svchost.exe -k ICService -p
vmicshutdown                                           C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
vmictimesync                                           C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
vmicvmsession                                          C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
vmicvss                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
VMnetDHCP                                              C:\Windows\SysWOW64\vmnetdhcp.exe
VMUSBArbService                                        "C:\Program Files (x86)\Common Files\VMware\USB\vmware-usbarbitrator64.exe"
VMware NAT Service                                     C:\Windows\SysWOW64\vmnat.exe
VSS                                                    C:\Windows\system32\vssvc.exe
VSStandardCollectorService150                          "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\DiagnosticsHub.Collection.Service\StandardCollector.Service.exe"
W32Time                                                C:\Windows\system32\svchost.exe -k LocalService
WaaSMedicSvc                                           C:\Windows\system32\svchost.exe -k wusvcs -p
WalletService                                          C:\Windows\System32\svchost.exe -k appmodel -p
WarpJITSvc                                             C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted
wbengine                                               "C:\Windows\system32\wbengine.exe"
WbioSrvc                                               C:\Windows\system32\svchost.exe -k WbioSvcGroup
Wcmsvc                                                 C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
wcncsvc                                                C:\Windows\System32\svchost.exe -k LocalServiceAndNoImpersonation -p
WdiServiceHost                                         C:\Windows\System32\svchost.exe -k LocalService -p
WdiSystemHost                                          C:\Windows\System32\svchost.exe -k LocalSystemNetworkRestricted -p
WdNisSvc                                               "C:\ProgramData\Microsoft\Windows Defender\platform\4.18.1910.4-0\NisSrv.exe"
WebClient                                              C:\Windows\system32\svchost.exe -k LocalService -p
Wecsvc                                                 C:\Windows\system32\svchost.exe -k NetworkService -p
WEPHOSTSVC                                             C:\Windows\system32\svchost.exe -k WepHostSvcGroup
wercplsupport                                          C:\Windows\System32\svchost.exe -k netsvcs -p
WerSvc                                                 C:\Windows\System32\svchost.exe -k WerSvcGroup
WFDSConMgrSvc                                          C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
WiaRpc                                                 C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
WinDefend                                              "C:\ProgramData\Microsoft\Windows Defender\platform\4.18.1910.4-0\MsMpEng.exe"
WinHttpAutoProxySvc                                    C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
Winmgmt                                                C:\Windows\system32\svchost.exe -k netsvcs -p
WinRM                                                  C:\Windows\System32\svchost.exe -k NetworkService -p
wisvc                                                  C:\Windows\system32\svchost.exe -k netsvcs -p
WlanSvc                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
wlidsvc                                                C:\Windows\system32\svchost.exe -k netsvcs -p
wlpasvc                                                C:\Windows\system32\svchost.exe -k LocalServiceNetworkRestricted -p
WManSvc                                                C:\Windows\system32\svchost.exe -k netsvcs -p
wmiApSrv                                               C:\Windows\system32\wbem\WmiApSrv.exe
WMPNetworkSvc                                          "C:\Program Files\Windows Media Player\wmpnetwk.exe"
workfolderssvc                                         C:\Windows\System32\svchost.exe -k LocalService -p
WpcMonSvc                                              C:\Windows\system32\svchost.exe -k LocalService
WPDBusEnum                                             C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted
WpnService                                             C:\Windows\system32\svchost.exe -k netsvcs -p
wscsvc                                                 C:\Windows\System32\svchost.exe -k LocalServiceNetworkRestricted -p
WSearch                                                C:\Windows\system32\SearchIndexer.exe /Embedding
wuauserv                                               C:\Windows\system32\svchost.exe -k netsvcs -p
WwanSvc                                                C:\Windows\system32\svchost.exe -k LocalSystemNetworkRestricted -p
XblAuthManager                                         C:\Windows\system32\svchost.exe -k netsvcs -p
XblGameSave                                            C:\Windows\system32\svchost.exe -k netsvcs -p
XboxGipSvc                                             C:\Windows\system32\svchost.exe -k netsvcs -p
XboxNetApiSvc                                          C:\Windows\system32\svchost.exe -k netsvcs -p
AarSvc_7214b                                           C:\Windows\system32\svchost.exe -k AarSvcGroup -p
BcastDVRUserService_7214b                              C:\Windows\system32\svchost.exe -k BcastDVRUserService
BluetoothUserService_7214b                             C:\Windows\system32\svchost.exe -k BthAppGroup -p
CaptureService_7214b                                   C:\Windows\system32\svchost.exe -k LocalService -p
cbdhsvc_7214b                                          C:\Windows\system32\svchost.exe -k ClipboardSvcGroup -p
CDPUserSvc_7214b                                       C:\Windows\system32\svchost.exe -k UnistackSvcGroup
ConsentUxUserSvc_7214b                                 C:\Windows\system32\svchost.exe -k DevicesFlow
CredentialEnrollmentManagerUserSvc_7214b               C:\Windows\system32\CredentialEnrollmentManager.exe
DeviceAssociationBrokerSvc_7214b                       C:\Windows\system32\svchost.exe -k DevicesFlow -p
DevicePickerUserSvc_7214b                              C:\Windows\system32\svchost.exe -k DevicesFlow
DevicesFlowUserSvc_7214b                               C:\Windows\system32\svchost.exe -k DevicesFlow
MessagingService_7214b                                 C:\Windows\system32\svchost.exe -k UnistackSvcGroup
OneSyncSvc_7214b                                       C:\Windows\system32\svchost.exe -k UnistackSvcGroup
PimIndexMaintenanceSvc_7214b                           C:\Windows\system32\svchost.exe -k UnistackSvcGroup
PrintWorkflowUserSvc_7214b                             C:\Windows\system32\svchost.exe -k PrintWorkflow
UnistoreSvc_7214b                                      C:\Windows\System32\svchost.exe -k UnistackSvcGroup
UserDataSvc_7214b                                      C:\Windows\system32\svchost.exe -k UnistackSvcGroup
WpnUserService_7214b                                   C:\Windows\system32\svchost.exe -k UnistackSvcGroup
```

**Comment by pal1000** [comment]
Nothing wrong is visible so far.  Open an elevated command prompt and run these commands. I want to know which fail:
````
REG DELETE HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Run /v RtkAudUService /f
net stop RtkAudioUniversalService
sc delete RtkAudioUniversalService
taskkill /f /im RtkAudUService64.exe
````
Note that these will break Realtek Audio Console. You'll have to run setup.cmd to fix it.

**Comment by julianxhokaxhiu** [comment]
I'll do it a bit later as I'm a little busy now. But I'll update this issue as soon as possible.

**Comment by julianxhokaxhiu** [comment]
I did a test of all the commands and all of them succeeded:
```
C:\Windows\system32>REG DELETE HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Run /v RtkAudUService /f
Operazione completata.

C:\Windows\system32>net stop RtkAudioUniversalService

Servizio Realtek Audio Universal Service arrestato.


C:\Windows\system32>sc delete RtkAudioUniversalService
[SC] DeleteService OPERAZIONI RIUSCITE

C:\Windows\system32>taskkill /f /im RtkAudUService64.exe
OPERAZIONE RIUSCITA: il processo "RtkAudUService64.exe" con il PID 8764 è stato terminato.
```

**Comment by pal1000** [comment]
I figured out what is wrong.  Your computer reached a state where Realtek Audio Universal Service usermode registration is missing.  It reached this state when debugging failure of elevation routine which was your original problem, which was fixed by #18.  Unfortunately setup.cmd didn't expect this scenario to happen so `net stop` and `sc delete` commands never ran and `taskkill /f` doesn't work on kernel modules so you were stuck with the kernel module being unstoppable.  The scenario on your computer that setup.cmd couldn't handle has already been resolved by my attempts to diagnose it so you shouldn't have more issues in the future. I am preparing a patch to handle this scenario in the future. Your issue share its root cause with #21 and another report from [station-drivers](https://station-drivers.com/index.php?option=com_kunena&view=topic&defaultmenu=860&Itemid=858&catid=18&id=17&lang=en&limitstart=912#2450).
